lib/drag_time_api_web/controllers/event_controller.ex

```diff
@@ -0,0 +1,53 @@
+defmodule DragTimeApiWeb.EventController do
+  use DragTimeApiWeb, :controller
+
+  alias DragTimeApi.Repo
+  alias DragTimeApi.Events
+  alias DragTimeApi.Events.Event
+
+  action_fallback DragTimeApiWeb.FallbackController
+
```

Comment:
What is this grouping of files? Controllers?

Reply:
The use DragTimeApiWeb is telling the controller to use the DragTimeApiWeb module. 

The alias section is a reference to other used modules within the controller. Setting alias DragTimeApi.Repo translates to Repo. So you don't have to call DragTimeApi.Repo within the controller, you can just call Repo.